### PR TITLE
Improve documentation references in `README`

### PR DESCRIPTION
**Pull request by mixmix**
In response to : https://github.com/entropyxyz/entropy-core/issues/1110

Wanted to offer this PR that makes it easier for others to get started with `entropy-core` :pray: :blue_heart:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,16 @@ Our blockchain node is written with [Substrate](https://substrate.io/) using [Su
 
 ## Documentation
 
-You can build the API documentation for Entropy by invoking `cargo doc --no-deps --open`.
+- High level introduction to Entropy: [docs.entropy.xyz](https://docs.entropy.xyz)
+- API documentation for this crate: [docs.rs/entroyp-ts](https://docs.rs/entropy-tss/latest/entropy_tss/index.html)
```
Comment (HCastano) [inline README.md:12]:
```suggestion
- API documentation for the `entropy-tss` crate: [docs.rs/entropy-tss](https://docs.rs/entropy-tss/latest/entropy_tss/index.html)
```

File: README.md [hunk 2]
```diff
@@ -8,7 +8,16 @@ Our blockchain node is written with [Substrate](https://substrate.io/) using [Su
...
+
+You can also build the API docs yourself:
+1. [Install the dependencies](#building-from-source)
+2. Invoke
+    ```bash
+    cargo doc --no-deps --open`
+    ```
+
 There is also [high level documentation for Entropy available here](https://docs.entropy.xyz).
```
Comment (HCastano) [inline README.md:21]: We can get rid of this line now

**Review by HCastano — APPROVED** [review]
Cool thanks

File: README.md (hunk 2 above)
Comment (mixmix) [reply README.md:21]:
```suggestion
```

**Comment by mixmix** [comment]
Made those changes. I'll leave you to merge @HCastano (not my repo!)

**Pull request merged**